SDK_versioned_docs/version-3.0.0/guides/04-fetching-prices.md

```diff
@@ -0,0 +1,143 @@
+---
+id: fetching-prices
+title: Fetching Spot Prices
+---
+
+## Fetching Token Prices with the SDK
+
+To calculate the spot price of any pool, you can call trivially call `token0Price` or `token1Price` on any Pool object (how nice!). However, in addition to learning a couple of simple steps needed to fetch token prices, you will also learn *how* the SDK calculates these quantities. Going through these calculations will hopefully provide the necessary context behind fixed point numbers, square roots, and difficult-to-understand variable names like `sqrtPriceX96`. :)
```

Comment:
would avoid "trivially" because if someone is confused and feeling underwhelmed, it might make them feel dumb.  maybe "simply" instead

Reply:
if Pool is a class type then should we code-case it?

Reply:
I don't quite follow the "However" sentence. Am I going to learn about token0Price in this guide? or not? 

maybe:  "This guide will teach you how to fetch the current market price of any token on Uniswap. First you will learn the simple method of calling `token0Price` or `token1Price`. Then you will peek under the hood and learn how the SDK calculates these quantities from <on-chain oracle data?>. Going through these calculations...rest is great"